librz/util/file.c

```diff
@@ -1181,6 +1181,32 @@ RZ_API bool rz_file_copy(const char *src, const char *dst) {
 	/* TODO: Use NO_CACHE for iOS dyldcache copying */
 #if HAVE_COPYFILE
 	return copyfile(src, dst, 0, COPYFILE_DATA | COPYFILE_XATTR) != -1;
+#elif HAVE_COPY_FILE_RANGE
+	/**
+	 * Unfortunately on FreeBSD the syscall
+	 * does not live up to the promise
+	 */
```

Comment:
remove this and add a smaller component in the meson.build file near the right line